feature/bgp/static_route_bgp_redistribution/otg_tests/static_route_bgp_redistribution_test/static_route_bgp_redistribution_test.go

```diff
@@ -1577,7 +1668,7 @@ func TestBGPStaticRouteRedistribution(t *testing.T) {
 			name:  "1.27.5 redistribute-ipv4-route-policy-as-prepend",
 			setup: func() { redistributeStaticRoutePolicyWithASN(t, dut, isV4) },
 			validate: func() {
-				validatePrefixASN(t, ate, isV4, atePort1.Name+".BGP4.peer", "192.168.10.0", []uint32{64512, 65499, 65499, 65499})
+				validatePrefixASN(t, ate, isV4, atePort1.Name+".BGP4.peer", "192.168.10.0", []uint32{65499, 65499, 65499, 64512})
```

Comment:
1.27.5_redistribute-ipv4-route-policy-as-prepend is failing at validatePrefixASN on Juniper device.

[static_route_bgp_redistribution_test.go:1307]
Prefix not updated with required as-path. Got [64512], want [65499 65499 65499 64512]

Reply:
@nsadhasivam I will check and update it.